sys/Makefile

```diff
@@ -6,16 +6,19 @@ CP		:= cp
 INSTALL		:= install
 RM		:= rm -rf
 MAKE		:= make
+MKDIR		:= mkdir
 
-.PHONY: all clean crts default_arm7 dldi_r4 install
+.PHONY: all clean crts arm7 dldi_r4 install
 
-all: crts default_arm7 dldi_r4
+all: crts arm7 dldi_r4
 
 crts:
 	$(MAKE) -C crts
 
-default_arm7: crts
-	$(MAKE) -C default_arm7
+arm7: crts
+	$(MAKE) -C arm7
+	$(MKDIR) -p default_arm7
+	$(CP) arm7/main_core/arm7_dswifi_maxmod.elf default_arm7/arm7.elf
```

Comment:
Ah, you also need to copy the COPYING file from the arm7 folder to the installed default_arm7 folder.